code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -695,4 +695,7 @@
 	description = "Fluorosulfuric acid is a an extremely corrosive chemical substance."
 	color = "#8E18A9" // rgb: 142, 24, 169
 	toxpwr = 2
-	acidpwr = 20
+	acidpwr = 42.0
```

Comment:
I would like a gif of someone being sprayed or gassed with the acid to see how quickly it kills.